nav2_costmap_2d/test/integration/obstacle_tests.cpp

```diff
@@ -554,3 +554,50 @@ TEST_F(TestNode, testDynParamsSetStatic)
   costmap->on_cleanup(rclcpp_lifecycle::State());
   costmap->on_shutdown(rclcpp_lifecycle::State());
 }
+
+
+class TestNodeWithoutUnknownOverwrite : public ::testing::Test
+{
+public:
+  TestNodeWithoutUnknownOverwrite()
+  {
+    node_ = std::make_shared<TestLifecycleNode>("obstacle_test_node");
+    node_->declare_parameter("track_unknown_space", rclcpp::ParameterValue(true));
+    node_->declare_parameter("lethal_cost_threshold", rclcpp::ParameterValue(100));
+    node_->declare_parameter("trinary_costmap", rclcpp::ParameterValue(true));
+    node_->declare_parameter("transform_tolerance", rclcpp::ParameterValue(0.3));
+    node_->declare_parameter("observation_sources", rclcpp::ParameterValue(std::string("")));
+    // MaxWithoutUnknownOverwrite
+    node_->declare_parameter("obstacles.combination_method", rclcpp::ParameterValue(2));
+  }
+
+  ~TestNodeWithoutUnknownOverwrite() {}
+
+protected:
+  std::shared_ptr<TestLifecycleNode> node_;
+};
+
+/**
+ * Test CombinationMethod::MaxWithoutUnknownOverwrite in ObstacleLayer.
+ */
+TEST_F(TestNodeWithoutUnknownOverwrite, testMaxWithoutUnknownOverwriteCombinationMethod) {
```

Comment:
Can you show that these same inputs with MaxOverwrite does change the unknown count?